src/sentry/integrations/slack/message_builder/event.py

```diff
@@ -1,22 +1,48 @@
+from typing import Any
+
+from sentry import features
 from sentry.integrations.slack.message_builder import SlackBody
 from sentry.integrations.slack.message_builder.base.block import BlockSlackMessageBuilder
 
+HEADER_MESSAGE = "Here are the commands you can use. Commands not working? Re-install the app!"
+DM_COMMAND_HEADER = "*Direct Message Commands:*"
+DM_COMMANDS_MESSAGE = "• `/sentry link`: Link your Slack account to Sentry \n • `/sentry unlink`: Unlink your Slack account from Sentry\n • `/sentry help`, `help`: View this list of commands"
```

Comment:
You should make a mapping of commands to descriptions that we can reuse and then do something like: 
```
DM_COMMANDS_MESSAGE = "\n".join((
	f"• `/sentry {command}`: {description}"
	for command, description in sorted(tuple(AVAILABLE_COMMANDS.items()))
))
```